azurerm/resource_arm_data_lake_store.go

```diff
@@ -0,0 +1,193 @@
+					string(account.Commitment500TB),
+					string(account.Commitment1PB),
+					string(account.Commitment5PB),
+				}, true),
+			},
+
+			"tags": tagsSchema(),
+		},
+	}
+}
+
+func resourceArmDateLakeStoreCreate(d *schema.ResourceData, meta interface{}) error {
+	client := meta.(*ArmClient).dataLakeStoreAccountClient
+	ctx := meta.(*ArmClient).StopContext
+
+	log.Printf("[INFO] preparing arguments for Azure ARM Date Lake Store creation.")
+
+	name := d.Get("name").(string)
+	location := azureRMNormalizeLocation(d.Get("location").(string))
+	resGroup := d.Get("resource_group_name").(string)
+	tier := d.Get("tier").(string)
+	tags := d.Get("tags").(map[string]interface{})
+
+	dateLakeStore := account.CreateDataLakeStoreAccountParameters{
+		Location: &location,
+		Tags:     expandTags(tags),
+		CreateDataLakeStoreAccountProperties: &account.CreateDataLakeStoreAccountProperties{
+			NewTier: account.TierType(tier),
+		},
+	}
+
+	future, err := client.Create(ctx, resGroup, name, dateLakeStore)
+	if err != nil {
+		return err
+	}
+
+	err = future.WaitForCompletion(ctx, client.Client)
+	if err != nil {
+		return err
+	}
+
+	read, err := client.Get(ctx, resGroup, name)
+	if err != nil {
+		return err
+	}
+	if read.ID == nil {
+		return fmt.Errorf("Cannot read Data Lake Store %s (resource group %s) ID", name, resGroup)
+	}
+
+	d.SetId(*read.ID)
+
+	return resourceArmDateLakeStoreRead(d, meta)
+}
+
+func resourceArmDateLakeStoreUpdate(d *schema.ResourceData, meta interface{}) error {
+	client := meta.(*ArmClient).dataLakeStoreAccountClient
+	ctx := meta.(*ArmClient).StopContext
+
+	name := d.Get("name").(string)
+	resourceGroup := d.Get("resource_group_name").(string)
+	newTags := d.Get("tags").(map[string]interface{})
+	newTier := d.Get("tier").(string)
+
+	props := account.UpdateDataLakeStoreAccountParameters{
+		Tags: expandTags(newTags),
+		UpdateDataLakeStoreAccountProperties: &account.UpdateDataLakeStoreAccountProperties{
+			NewTier: account.TierType(newTier),
+		},
+	}
+
+	future, err := client.Update(ctx, resourceGroup, name, props)
+	if err != nil {
+		return fmt.Errorf("Error issuing update request for Data Lake Store %q (Resource Group %q): %+v", name, resourceGroup, err)
+	}
+
+	err = future.WaitForCompletion(ctx, client.Client)
+	if err != nil {
+		return fmt.Errorf("Error waiting for the update of Data Lake Store %q (Resource Group %q) to commplete: %+v", name, resourceGroup, err)
+	}
+
+	return resourceArmDateLakeStoreRead(d, meta)
+}
+
+func resourceArmDateLakeStoreRead(d *schema.ResourceData, meta interface{}) error {
+	client := meta.(*ArmClient).dataLakeStoreAccountClient
+	ctx := meta.(*ArmClient).StopContext
+
+	id, err := parseAzureResourceID(d.Id())
+	if err != nil {
+		return err
+	}
+	resourceGroup := id.ResourceGroup
+	name := id.Path["accounts"]
+
+	resp, err := client.Get(ctx, resourceGroup, name)
+	if err != nil {
+		if utils.ResponseWasNotFound(resp.Response) {
+			d.SetId("")
+			fmt.Print("Oh no nothing was found")
```

Comment:
Can you format the not found message to something like...
`log.Printf("[WARN] DataLakeStore '%s' was not found (resource group '%s')", name, resourcGroup)`